hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobExecutionService.java

```diff
@@ -82,13 +83,13 @@
         this.jobRepository = jobRepository;
     }
 
-    public ClassLoader getClassLoader(JobConfig config, long jobId) {
+    public ClassLoader getClassLoader(NodeEngine nodeEngine, JobConfig jobConfig, long jobId) {
```

Comment:
there is already `nodeEngine` as a field, why not use it directly?

Reply:
good point :)